data_diff/databases/database_types.py

```diff
@@ -1,11 +1,32 @@
+from uuid import UUID
 from abc import ABC, abstractmethod
-from typing import Sequence, Optional, Tuple, Union, Dict
+from typing import Sequence, Optional, Tuple, Union, Dict, Any
 from datetime import datetime
 
 from runtype import dataclass
 
+
+class ArithUUID(UUID):
```

Comment:
This is sweet

Reply:
Yeah, the magic of Python. It wouldn't be possible in most languages.